src/clr-angular/forms/checkbox/checkbox-container.ts

```diff
@@ -15,12 +15,13 @@ import { LayoutService } from '../common/providers/layout.service';
 import { NgControlService } from '../common/providers/ng-control.service';
 
 @Component({
-  selector: 'clr-checkbox-container',
+  selector: 'clr-checkbox-container,clr-toggle-container',
   template: `
     <ng-content select="label"></ng-content>
     <label *ngIf="!label && addGrid()"></label>
     <div class="clr-control-container" [class.clr-control-inline]="clrInline" [ngClass]="controlClass()">
       <ng-content select="clr-checkbox-wrapper"></ng-content>
+      <ng-content select="clr-toggle-wrapper"></ng-content>
```

Comment:
Nitpick: I'd go with `<ng-content select="clr-checkbox-wrapper, clr-toggle-wrapper"></ng-content>` to better show that we're really using them in the same way. But I don't feel strongly about this.

Reply:
Sure, updated.